docs/solana/output.rst

```diff
@@ -1,9 +1,279 @@
+
+* Appear at the beginning of the trace
+* Represent the starting state which leads to a violation
+* Format: ``CVT_nondet_u64: <value>``
+
+Method Calls
+**********
+
+* Automatically logged
+* Show the sequence of function calls
+* To see parameter values, you need to munge the original code with CVT-style print statements
+
+Variable Values
+************
+
+Printed using the special macro ``cvt_cex_print_u64!()``. For named printing, you can add the following macro to your ``utils.rs``, this will print the name of the variable your passing. 
+
+.. code-block:: rust
+
+    macro_rules! nice_print {
+        ($e:expr) => {
+            cvt_cex_print_u64!($e, $e)
+        };
+    }
+    pub(crate) use nice_print;
+
+
+.. _example: https://prover.certora.com/output/696612/daa2d29059234c459ab0d2c7f6784cde?anonymousKey=cddfa55db76600ea920eea729a454cde61fe278e 
+
+Let's look at a concrete example_:
+*********************************
+
+.. code-block:: rust
+
+    pub fn is_greater_than_100(amount: u64) -> bool {
+        amount > 100
+    }
+
+    #[rule]
+    pub fn rule_fail_call_trace() {
+        let amount1: u64 = nondet();
+        let amount2: u64 = nondet();
+
+        let is_great = is_greater_than_100(amount1);
+        cvt_assume!(is_great);
+        let is_great2 = is_greater_than_100(amount2);
+        cvt_assume!(amount2 != 10);
+        
+        nice_print!(is_great);
+        nice_print!(amount1);
+        nice_print!(is_great2);
+        nice_print!(amount2);
+        
+        cvt_assert!(amount1 < 100);
+        cvt_assert!(amount2 < 100);
+    }
+
+When this rule fails, the prover generates a counterexample that looks like this:
+
+.. container:: call-trace-analysis
+   :name: call-trace-example
+      
+   .. image:: img/call_trace_detail.png
+        :alt: A call trace example
+        :align: right
+        :width: 300px
+
+      Call trace output showing counterexample values
+
+   Let's analyze this counterexample:
+
+   1. **Nondet Values**: 
+
+        * The prover chose ``101`` for ``amount1`` and implicitly ``2`` for ``amount2``
+        * These values are chosen to demonstrate the violation of our assertions
+        * ``<?>`` means that the value is not important for the counterexample 
+
+   2. **Variable States**:
+
+        * ``is_great`` is assumed, thus ``1`` (true) as ``amount1`` (101) is indeed greater than 100
+        * ``amount1`` is 101, which satisfies our assumption ``cvt_assume!(is_great)``
+        * ``is_great2`` is ``0`` (false) because ``amount2`` (2) is not greater than 100
+        * ``amount2`` is 2, which satisfies our assumption ``cvt_assume!(amount2 != 10)``
+
+   3. **Assertion Failure**:
+
+        * The assertion ``cvt_assert!(amount1 < 100)`` fails because we have a contradiction:
+            * We assumed ``is_great`` is true, which means ``amount1 > 100``
+            * But then we assert ``amount1 < 100``
+        * This is impossible to satisfy, hence the counterexample
```

Comment:
@phreppo I think we should add a few lines to explain the JTS feature here. Only caveat: we need to resubmit [the job used above](https://prover.certora.com/output/696612/daa2d29059234c459ab0d2c7f6784cde?anonymousKey=cddfa55db76600ea920eea729a454cde61fe278e), @paminacert can you please share the source code with Francesco for him to resubmit it including the uploaded source code?